src/main/resources/org/sagebionetworks/web/client/view/ChangeUsernameViewImpl.ui.xml

```diff
@@ -17,20 +14,16 @@
 							carefully choose yours.
```

Comment:
I had no idea it was possible for a user to have no username, and that we have UI for such a case.